docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx

```diff
@@ -6,8 +6,12 @@ description: Configure the Next.js cache used for storing and revalidating data.
 In Next.js, the [default cache handler](/docs/app/building-your-application/data-fetching/caching) uses the filesystem cache. This requires no configuration, however, you can customize the cache handler by using the `incrementalCacheHandlerPath` field in `next.config.js`.
 
 ```js filename="next.config.js"
+const path = require('path')
+
 module.exports = {
-  incrementalCacheHandlerPath: './cache-handler.js',
+  experimental: {
+    incrementalCacheHandlerPath: path.resolve('./cache-handler.js'),
```

Comment:
Could this be `require.resolve('./cache-handler.js')` instead?

Reply:
Yes, it could be

Reply:
updated